_posts/0000-01-02-s-harrison.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Welcome! This is the 2nd slide for this slideshow
```

Comment:
![thank-you-cheers](https://user-images.githubusercontent.com/89761881/133715559-180035db-e0ea-45cf-9724-1b245ff3650c.gif)